src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -0,0 +1,309 @@
+import { html, LitElement } from "@polymer/lit-element";
+import { classMap } from "lit-html/directives/classMap.js";
+import roundSliderStyle from "round-slider/dist/roundslider.min.css";
+
+import "../../../components/ha-card.js";
+import "../../../components/ha-icon.js";
+import "../../../resources/jquery.roundslider";
+
+import { HomeAssistant } from "../../../types.js";
+import { HassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+
+const thermostatConfig = {
+  radius: 150,
+  min: 50,
+  max: 90,
+  step: 1,
+  circleShape: "pie",
+  sliderType: "min-range",
+  value: 70,
+  startAngle: 315,
+  width: 5,
+  lineCap: "round",
+  handleSize: "+10",
+  showTooltip: false,
+};
+
+const modeIcons = {
+  auto: "mdi:autorenew",
+  heat: "hass:fire",
+  cool: "hass:snowflake",
+  off: "mdi:fan-off",
```

Comment:
`hass:`

Reply:
There was a bug that we didn't extract icons from typescript files but that was fixed by @iantrich

Reply:
Yea I saw that. Forgot about changing thank you